pkg/component/component.go

```diff
@@ -135,8 +138,9 @@ func GetCurrent(client *occlient.Client, applicationName string, projectName str
 }
 
 // PushLocal start new build and push local dir as a source for build
```

Comment:
Please update comment, as it is no longer true. Now it can push the single binary file on top of a directory.